### PR TITLE
Enable UTM params feature

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-frontend/issues/9616

### CODE REVIEW

File: tests/unit/core/test_addonManager.js [hunk 1]
```diff
@@ -221,7 +221,8 @@ describe(__filename, () => {
       });
 
       sinon.assert.calledWith(fakeMozAddonManager.createInstall, {
-        url,
```
Comment (willdurand) [inline tests/unit/core/test_addonManager.js:224]: The test cases I fixed here are obsolete-ish. All we need is to fix https://github.com/mozilla/addons-frontend/issues/9556 so ignore the changes here 😬

File: tests/unit/amo/components/TestAddonMoreInfo.js [hunk 2]
```diff
@@ -35,8 +35,11 @@ describe(__filename, () => {
   });
 
   function render({ location, ...props } = {}) {
+    const _config = getFakeConfig({ enableFeatureUseUtmParams: false });
```
Comment (bobsilverberg) [inline tests/unit/amo/components/TestAddonMoreInfo.js:38]: If the plan is to turn the feature on, wouldn't it make more sense for the test to assume the feature is on, and not off? Are you just doing this to keep the patch small, and deferring updating the test to a different patch?

File: tests/unit/amo/components/TestSearchResult.js [hunk 3]
```diff
@@ -92,9 +92,10 @@ describe(__filename, () => {
   });
 
   it('links the heading to the detail page with a source', () => {
+    const _config = getFakeConfig({ enableFeatureUseUtmParams: false });
```
Comment (bobsilverberg) [inline tests/unit/amo/components/TestSearchResult.js:95]: If the plan is to turn the feature on, wouldn't it make more sense for the test to assume the feature is on, and not off? Are you just doing this to keep the patch small, and deferring updating the test to a different patch?

**Review by bobsilverberg — APPROVED** [review]
Looks good, thanks @willdurand. I had a couple of questions about the tests.

r+wc

File: tests/unit/amo/components/TestAddonMoreInfo.js (hunk 2 above)
Comment (willdurand) [reply tests/unit/amo/components/TestAddonMoreInfo.js:38]: Yeah, the plan is to remove the feature flag once this is live.

**Pull request merged**